lib/matplotlib/axes/_base.py

```diff
@@ -3242,9 +3242,10 @@ def ticklabel_format(self, *, axis='both', style='', scilimits=None,
         axis : {'x', 'y', 'both'}, default: 'both'
             The axis to configure.  Only major ticks are affected.
 
-        style : {'sci', 'scientific', 'plain'}
+        style : {'sci', 'scientific', 'plain'} or None, default: None
```

Comment:
```suggestion
        style : {'sci', 'scientific', 'plain'}
```
I think this is more consistent with the other parameters of this method that default to `None`.

Reply:
I'm going to commit your suggestion. Quicker this way!